circleci/resource_circleci_environment_variable_test.go

```diff
@@ -12,48 +12,47 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
-func TestCircleCIEnvironmentVariableOrganizationNotSet(t *testing.T) {
+func TestAccCircleCIEnvironmentVariableOrganizationNotSet(t *testing.T) {
 	project := "TEST_" + acctest.RandString(8)
 	envName := "TEST_" + acctest.RandString(8)
 
 	resource.Test(t, resource.TestCase{
 		PreCheck: func() {
-			testPreCheck(t)
+			testAccPreCheck(t)
 		},
-		Providers:  resourceOrgTestProviders,
-		IsUnitTest: true,
```

Comment:
Ah, I think that was my bad in a previous PR. I forgot to remove it, sorry

Reply:
No worries, I missed that as well in the review and also when fixing it was under my nose the whole time but it took me a while to spot it ahah